services/checks/checks.go

```diff
@@ -42,14 +42,15 @@ import (
 
 	"github.com/percona/pmm-managed/models"
 	"github.com/percona/pmm-managed/services"
+	"github.com/percona/pmm-managed/utils/envvars"
 )
 
 const (
-	defaultHost            = "check.percona.com:443"
 	defaultRestartInterval = 24 * time.Hour
 	defaultStartDelay      = time.Minute
 
 	// Environment variables that affect checks service; only for testing.
+	// FIXME remove https://jira.percona.com/browse/SAAS-360
 	envHost            = "PERCONA_TEST_CHECKS_HOST"
```

Comment:
```suggestion
	envHost            = "PERCONA_TEST_CHECKS_HOST" // FIXME remove https://jira.percona.com/browse/SAAS-360
```

To make this a comment for this particular line, not the whole section. Please do the same in other files

Reply:
Done